pkgs/desktops/pantheon/apps/switchboard-plugs/power/default.nix

```diff
@@ -54,13 +54,6 @@ stdenv.mkDerivation rec {
     switchboard
   ];
 
-  patches = [
-    (substituteAll {
-      src = ./dpms-helper-exec.patch;
-      elementary_dpms_helper = elementary-dpms-helper;
```

Comment:
We can remove dpms-helper from the rest of this derivation.

And maybe from Nixpkgs altogether if nothing else depends on it.

https://github.com/elementary/switchboard-plug-power/pull/163